src/miscellaneous.py

```diff
@@ -0,0 +1,154 @@
+import sys, json
+
+sys.path.append("..")
+from app import *
+from src.util import *
+from flask import Flask, abort, request, jsonify, render_template
+from flask_restful import reqparse, abort, Api, Resource
+from bson.json_util import dumps, loads
+
+parser = reqparse.RequestParser()
+parser.add_argument("tag")
+parser.add_argument("display")
+
+
+class Hashtag(Resource):
+    def get(self):
+        """Gets the hastag list.
+
+        Returns:
+            A list of hashtag details
+
+        Example:
+        [
+            {
+                "name": "example_tag_1",
+                "is_display": 1
+            },
+            {
+                "name": "example_tag_2",
+                "is_display": 1
+            },
+            {
+                "name": "example_tag_3",
+                "is_display": 0
+            }
+        ]
+        """
+        request.get_json()
+        args = parser.parse_args()
+        print(args)
+        if args["display"] and args["tag"]:
```

Comment:
is there a reason why we don't want to allow the get method with both arguments?

Reply:
I think we are either getting a tag with specific name to see if the is_display is true, or parsing a display argument to get a list of all hashtags with specified display bool. I'm not sure if we need to use a GET if we know both tag name and display...